Shopify/ruby-lsp
# Unify ruby-lsp-doctor with ruby-lsp command

### Motivation

Improve discoverability of the troubleshooting tool.

### Implementation

I chose not to deprecate rather than fully remove since we have a number of issues mentioning the tool. We can revisit in future.

### Automated Tests

None

### Manual Tests

* Run `./exe/ruby-lsp -d` and verify the checks are performed.
* Run `./exe/ruby-lsp-doctor` and verify it prints a deprecation notice.
